app/controllers/application_records_controller.rb

```diff
@@ -22,14 +22,18 @@ def create
   def csv_export
     start_date = parse_american_date(params.require :start_date)
     end_date = parse_american_date(params.require :end_date)
-    @records = ApplicationRecord.between(start_date, end_date)
+    params.require :department_id
+    @records = ApplicationRecord.in_department(params[:department_id])
+               .between(start_date, end_date)
     render 'csv_export.csv.erb', layout: false
   end
 
   def eeo_data
     start_date = parse_american_date(params.require :eeo_start_date)
     end_date = parse_american_date(params.require :eeo_end_date)
-    @records = ApplicationRecord.eeo_data(start_date, end_date)
+    params.require :department_id
+    @records = ApplicationRecord
+               .eeo_data(start_date, end_date, params[:department_id])
```

Comment:
Stylistically I think I prefer the look of line-splitting the params rather than like this.